internal/cmd/skupper/connector/kube/connector_create_test.go

```diff
@@ -910,6 +910,7 @@ func TestCmdConnectorCreate_WaitUntil(t *testing.T) {
 	}
 
 	for _, test := range testTable {
+		test := test
```

Comment:
why is this needed?

Reply:
It is quite tricky. These redeclare the test variable in the correct scope - a quirk of go that is changing in the language with 1.22 https://go.dev/blog/loopvar-preview

It is one of the reasons I prefer to not use t.Parallel(), but felt it was worth it in these cases.